core/conversion/evaluators/aten.cpp

```diff
@@ -141,6 +141,26 @@ auto aten_registrations TORCHTRT_UNUSED =
                auto out_tensor = torch::zeros(args.at(n->input(0)).unwrapToIntList().vec(), options);
                return out_tensor;
              }})
+        .evaluator(
+            {c10::Symbol::fromQualString("aten::new_zeros"),
+             // aten::new_zeros(Tensor self, int[] size, *, ScalarType? dtype=None, Layout? layout=None,
+             // Device? device=None, bool? pin_memory=None) -> (Tensor)
+             [](const torch::jit::Node* n, kwargs& args) -> c10::optional<torch::jit::IValue> {
+               auto options = torch::TensorOptions().layout(torch::kStrided).device(torch::kCUDA);
+
+               // Unpack tensor input
+               auto input_dtype = args.at(n->input(0)).unwrapToTensor().dtype().toScalarType();
+
+               // If dtype is not specified, use those from self
```

Comment:
As per torch documentation (https://pytorch.org/docs/1.13/generated/torch.Tensor.new_zeros.html?highlight=new_zeros#torch.Tensor.new_zeros), device should also be inherited from the input tensor. In this case, is it still safe to set the device to CUDA, or should the tensor default to CUDA regardless?